selenium/src/test/java/au/org/emii/portal/utils/SeleniumUtil.java

```diff
@@ -75,26 +83,33 @@ public int validateInvalidLinks() {
     }
 
     public int verifyURLStatus(String URL, int invalidLinksCount) {
-        log.info("Validating URL " + URL);
-        //Validating the contact us email
-        if (URL.startsWith("mailto:info@aodn.org.au?subject=")) {
-            return invalidLinksCount;
-        }
-        try {
-
-            HttpGet request = new HttpGet(URL);
-            HttpResponse response = getHttpclient().execute(request);
-            // verifying response code and The HttpStatus should be 200 if not,
-            // increment invalid link count
-            ////We can also check for 404 status code like response.getStatusLine().getStatusCode() == 404
-            if (response.getStatusLine().getStatusCode() != 200) {
-                invalidLinksCount++;
-                log.error("Invalid URL " + URL);
+        if (StringUtils.isNotBlank(URL)) {
+            log.info("Validating URL " + URL);
+            //Validating the contact us email
+            if (URL.startsWith("mailto:info@aodn.org.au?subject=")) {
```

Comment:
Is it enough to look for mailto links? eg: `if (URL.startsWith("mailto")) {`

Trying to avoid embeding `info@aodn.org.au`